plotUtils/ModuleMapper.py

```diff
@@ -4,264 +4,262 @@ class ModuleMapper(dict):
     string_to_hw = {}
     string_to_sw = {}
     hw_to_string = {}
-    hw_to_sw ={}
-    sw_to_hw ={}
+    hw_to_sw = {}
+    sw_to_hw = {}
     sw_to_string = {}
     global_channel_map = {}
+
     def __init__(self):
 
-        ModuleMapper.hw_to_sw["F0H0"] = "ly1_m0" ;
```

Comment:
is the `;` at the end of the line relevant?

Reply:
no, it's not.